### ISSUE TITLE
Failed to read 10MB+ SVG

### ISSUE THREAD

**Issue by miyanokomiya** [imgproxy/imgproxy#1002]
## Issue
I ran into this error message when I tried to read 10MB+ SVG.
```
glib: XML parse error: Error domain 1 code 77 on line 1 column 10003742 of data: Premature end of data in tag g line 1\n\n\n
```

`vips_svgload` seems to have 10MB limitation by default.
https://github.com/libvips/libvips/blob/master/libvips/foreign/svgload.c#L1023

I believe we have to pass `unlimited` flag in order to disable this limitation.
(to here?)
https://github.com/imgproxy/imgproxy/blob/master/vips/vips.c#L93

## Request
Is there any way to pass the flag to `vips`? If not, I'd like to request a feature to do that via env value.

## Ref
- https://github.com/lovell/sharp/issues/2195#issuecomment-624862257
- https://github.com/lovell/sharp/blob/86acf2460e729ecb287d69196ff51ee0947b25cc/src/common.cc#L283

**Comment by DarthSim** [comment]
Added [IMGPROXY_SVG_UNLIMITED](https://docs.imgproxy.net/latest/configuration/options#IMGPROXY_SVG_UNLIMITED) config to the `latest` build.

**Comment by DarthSim** [comment]
The feature is released in `v3.24.0`
